### DISCUSSION TITLE
custom base path deployments

### DISCUSSION THREAD

**Discussion by marioweid** [langfuse/langfuse#2870] [Ideas]
### Describe the bug

Today i tried to run langfuse behind a reverse proxy under the path https://mydomain.com/langfuse.
I thought the NEXTAUTH_URL is the root URL for the web interface and that all requests for resources are getting resolved by that path. Unfortunately that's not the case and the application is still trying to resolve local paths like this f.e. on startup: https://mydomain.com/_next/...

My env vars are like this:
NEXTAUTH_URL=https://mydomain.com/langfuse
SALT=salt
HOSTNAME=0.0.0.0

I checked the docs, the issues and the source code and found no way to resolve this issue or set the root url.
Is there a way to run langfuse behind a reverse proxy and tell the application that it should search for stuff under the given root url f.e. https://mydomain.com/langfuse/.next/ ... ?

### To reproduce

1. Have a reverse proxy running f.e. nginx or application gateway in azure.
2. have a langfuse container running with the NEXTAUTH_URL=https://mydomain.com/langfuse
3. Configure the reverse proxy to direct all traffic unter /langfuse to the langfuse container

### Are you interested to contribute a fix for this bug?

No

**Comment by marcklingen** [comment]
currently deployments with a custom basePath are not supported, the feature request is tracked here: https://github.com/orgs/langfuse/discussions/2400

Open for contributions on this and please add your plus one on the feature request
